lib/dash/content_protection.js

```diff
@@ -327,6 +329,18 @@ shaka.dash.ContentProtection.parseElements_ = function(elems) {
           }
         }
 
+        // if there is no pssh block and keySystem is Playready,
+        // build Pssh block for Playready tags and, so MediaKeySession can be
+        // launched without waiting encrypted event.
+        var ContentProtection = shaka.dash.ContentProtection;
+        if (!psshs.length && ContentProtection.defaultKeySystems_[schemeUri] ===
```

Comment:
Alternatively, I can move this down and I can set pssh uint8array to 'init' directly and so we would not need decode and encode again. However, it would be weird for me.